appholder/src/main/java/com/android/mdl/app/document/DocumentManager.kt

```diff
@@ -338,7 +338,13 @@ class DocumentManager private constructor(private val context: Context) {
                 idsSelf,
                 signature
             )
-            .putEntryBoolean(DocumentData.AAMVA_NAMESPACE, "real_id", idsSelf, dData.getValueBoolean("real_id"))
+            .putEntryInteger(DocumentData.MDL_NAMESPACE, "sex", idsSelf, dData.getValueString("sex").toLong())
```

Comment:
super nitpick: formatting is inconsistent

But I think your new formatting is better :)

Reply:
I'll take that - thanks for the review!